### PR TITLE
Added firebase development config

### LINKED ISSUE #252: Add development firebase config file for iOS & Android
provide a non-production config file for both iOS and Android and add instruction to README

### PR DESCRIPTION
**Pull request by guytepper**
onboard new contributors faster by including a development firebase config, which they'll have to rename during the project installation. 

closes #252

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,11 @@ If this is your first time, check out the [official set up guide](https://reactn
 
 - Fork the repo and clone to your machine.
 - Run `yarn install`
-- Run `cd ios && pod install`
-- Run the app with `npm run ios`
+- Run `npm run rename-dev-configs` to rename the firebase development configs
+- Run `npm run start` to open the metro bundler
+- Run the app with `npm run ios` (for opening the iPhone simulator) or `npm run android` (for the Android simulator).
```
Comment (planecore) [inline README.md:27]:
```suggestion
- Run `yarn rename-dev-configs` to rename the firebase development configs
- Run `yarn start` to start metro dev server
- Run the app with `yarn ios` (to open the iPhone simulator) or `yarn android` (for the Android simulator).
```

File: package.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
     "adb": "adb reverse tcp:9090 tcp:9090 && adb reverse tcp:3000 tcp:3000 && adb reverse tcp:9001 tcp:9001 && adb reverse tcp:8081 tcp:8081",
     "postinstall": "patch-package; node ./bin/postInstall;",
     "build-ios": "react-native bundle --entry-file index.js --platform ios --dev false --bundle-output ios/main.jsbundle --assets-dest ios",
-    "build-android": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res"
+    "build-android": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res",
+    "rename-dev-configs": "cp ios/GoogleService-Info.development.plist ios/GoogleService-Info.plist; cp android/app/google-services.development.json android/app/google-services.json"
```
Comment (planecore) [inline package.json:26]: I don't think it'll work on Windows machines

**Review by planecore — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (guytepper) [reply package.json:26]: we can offer that..
Screenshot 2023-06-14 at 7 44 48

but I think it's going too far. let's offer the unix script and add a note for windows user that they should rename the files manually?

Comment (planecore) [reply package.json:26]: sounds good

**Review by planecore — APPROVED** [review]

**Pull request merged**